R/executeDqChecks.R

```diff
@@ -81,7 +83,7 @@ executeDqChecks <- function(connectionDetails,
                             fieldCheckThresholdLoc = "default",
                             conceptCheckThresholdLoc = "default") {
   # Check input -------------------------------------------------------------------------------------------------------------------
-  if (!any(class(connectionDetails) %in% c("connectionDetails", "ConnectionDetails"))) {
+  if (!sqlOnly && !any(class(connectionDetails) %in% c("connectionDetails", "ConnectionDetails"))) {
```

Comment:
the connectiondetails object should still be valid in `sqlOnly` mode, right, even though we don't use it connect to the DB? since we use it to determine the dbms?


... that said I don't love the idea of using a connectiondetails just to get the dbms for sqlOnly mode. maybe we can add a dbms parameter that's required if sqlOnly is true, and make connectionDetails optional for sqlOnly?

Reply:
I'll take a look.  Yes, in the sample, dbms is part of connectionDetails even if it is fake.  I agree with your concern - I'll see if it is easy to go that route (making dbms required and connectionDetail optional if using sqlOnly).

Reply:
adding @dbms would require a brand new parameter.  I stuck with connectionDetails  but reverted line 84.

Reply:
OK yeah I think it's fine to leave as-is for now, either way is somewhat non-ideal so think it's best to keep it simpler.